R/parsedate-package.r

```diff
@@ -79,8 +80,17 @@ yj <- function(x) as.POSIXct(x, format = "%Y %j", tz = "UTC")
 #'
 #' # But not for this, because this is ISO 8601
 #' parse_date("2014")
+#'
+#' # Handle vectors and empty input
+#' parse_date(c("2014","2015","","2016"))
 
 parse_date <- function(dates, approx = TRUE) {
+  result <- rep(as.POSIXct(NA), length=length(dates))
+  result[dates != ""] <- parse_date_f(dates[dates != ""])
```

Comment:
You are not passing `approx`.

To avoid this kind of mistakes, I think it is easier not to create an extra function.